snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/SnomedRestFixtures.java

```diff
@@ -226,14 +226,14 @@ public static Json createRelationshipRequestBody(String sourceId, String typeId,
 		return createRelationshipRequestBody(sourceId, typeId, destinationId, moduleId, characteristicTypeId, 0);
 	}
 
-	public static Json createRelationshipRequestBody(String sourceId, String typeId, String destinationId, String moduleId, String characteristicTypeId, int group) {
+	public static Json createRelationshipRequestBody(String sourceId, String typeId, String destinationId, String moduleId, String characteristicTypeId, int relationshipGroup) {
 		return Json.object(
 			"moduleId", moduleId,
 			"sourceId", sourceId,
 			"typeId", typeId,
 			"destinationId", destinationId,
 			"characteristicTypeId", characteristicTypeId,
-			"group", group
+			"relationshipGroup", relationshipGroup
```

Comment:
Yeah, I think there was a reason not to do it but now not so sure about it. Since this is not blocking the merge of the PR, let's pick this up after merge.